src/lib/model/OutboundTransfersModel.js

```diff
@@ -481,7 +483,7 @@ class OutboundTransfersModel {
                     this._logger.push({ fulfil }).log('Transfer fulfil received');
                     this.data.fulfil = fulfil;
 
-                    if(this._checkIlp && !this._ilp.validateFulfil(fulfil.fulfilment, this.data.quoteResponse.condition)) {
+                    if (this._checkIlp && !this._ilp.validateFulfil(fulfil.fulfilment, this.data.quoteResponse.condition)) {
```

Comment:
please dont change whitespace unless there is a very good reason to do so.

Reply:
Deleted